apps/catalogue/src/store/query/cohortDetails.gql

```diff
@@ -0,0 +1,176 @@
+query Cohorts($pid: String) {
```

Comment:
I think it would be better to generate this query? Now everytime data model changes you need to change it here also.

Reply:
The 'view' is a specific view on a specific model, changing the model means making decisions about changing the view.